CrossSectionAnalysis/CrossSectionAnalysis.py

```diff
@@ -1810,10 +1757,10 @@ def test_CrossSectionAnalysis1(self):
 SURFACE_AREA_STENOSIS_ARRAY_NAME = _("Stenosis by surface area")
 DIAMETER_STENOSIS_ARRAY_NAME = _("Stenosis by diameter (CE)")
 
-MIS_DIAMETER = "MIS_DIAMETER"
-CE_DIAMETER = "CE_DIAMETER"
-CROSS_SECTION_AREA = "CROSS_SECTION_AREA"
-WALL_CE_DIAMETER = "WALL_CE_DIAMETER"
-WALL_CROSS_SECTION_AREA = "WALL_CROSS_SECTION_AREA"
-DIAMETER_STENOSIS = "DIAMETER_STENOSIS"
-SURFACE_AREA_STENOSIS = "SURFACE_AREA_STENOSIS"
+MIS_DIAMETER_PLOT_SERIES_COLOR = (0.0, 0.6, 1.0)
```

Comment:
There are several chart color tables that contain colors that are supposed to work well for charts. It would be nicer to use those.

```
rgba=[0,0,0,0]
getNode('DarkBrightChartColors').GetColor(0, rgba)
print(rgba)
```